c7n/resources/resource_map.py

```diff
@@ -12,7 +12,8 @@
   "aws.apigwv2": "c7n.resources.apigw.ApiGwV2",
   "aws.apigwv2-stage": "c7n.resources.apigw.ApiGatewayV2Stage",
   "aws.appmesh-mesh": "c7n.resources.appmesh.AppmeshMesh",
-  "aws.appmesh-virtual-gateway": "c7n.resources.appmesh.AppmeshVirtualGateway",
```

Comment:
We can leave both forms of the resource name for backward compatibility.

Reply:
Nice - do i need to change?

Reply:
Yeah I think reverting the delete should do, sorry GitHub yelled at me when I tried to make a suggestion on a deleted line 😅

Reply:
done